docs/api.md

```diff
@@ -1083,6 +1085,65 @@ by the following errors
   Feature not enabled
 
 
+#### GET /account/device/messages
+
+:lock: HAWK-authenticated with session token
+<!--begin-route-get-accountdevicemessages-->
+Retrieve stored messages for the current device.
+The query parameters will be passed as-is to the Pushbox service: it is
+your responsability to set `index` and `limit`.
+The response is echoed from the Pushbox service.
+<!--end-route-get-accountdevicemessages-->
+
+##### Query parameters
+
+* `service`: *string, valid('sendtab'), required*
+
+  <!--begin-query-param-get-accountdevicemessages-service-->
+  
+  <!--end-query-param-get-accountdevicemessages-service-->
+
+* `index`: *string, optional*
+
+  <!--begin-query-param-get-accountdevicemessages-index-->
+  
+  <!--end-query-param-get-accountdevicemessages-index-->
+
+* `limit`: *number, optional*
+
+  <!--begin-query-param-get-accountdevicemessages-limit-->
+  
+  <!--end-query-param-get-accountdevicemessages-limit-->
+
+
+#### POST /account/device/messages
```

Comment:
nit: this feels like it fits better under "/account/devices/" rather than "/account/device", since the later currently implies an action that affects the current device while this affects some other device.